packages/shared/src/components/UserSettingsButton.tsx

```diff
@@ -4,14 +4,14 @@ import LayoutIcon from './icons/Layout';
 import { HeaderButton } from './buttons/common';
 import { SimpleTooltip } from './tooltips/SimpleTooltip';
 
-const FeedSettingsModal = dynamic(
+const UserSettingsModal = dynamic(
   () =>
     import(
-      /* webpackChunkName: "feedSettingsModal" */ './modals/FeedSettingsModal'
+      /* webpackChunkName: "UserSettingsModal" */ './modals/UserSettingsModal'
```

Comment:
It's not documented, but we should keep camelCase webpack chunk names, as suggested best practice.
I noticed we use it intertwined with PascalCase, but let's set for camelCase.

Reply:
Ah, I see. This was copy-pasted, thought it was alright to have it in PascalCase. I shall use camelCase in the future then.